mmdet/models/necks/pafpn.py

```diff
@@ -0,0 +1,200 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import xavier_init
+
+from mmdet.core import auto_fp16
+from mmdet.ops import ConvModule
+from ..registry import NECKS
+
+
+@NECKS.register_module
+class PAFPN(nn.Module):
```

Comment:
Could we inherit from FPN to reuse the FPN part of the model? It should reduce some duplicate code.